src/tmx/Messages/include/SensorDetectedObject.h

```diff
@@ -0,0 +1,45 @@
+#ifndef INCLUDE_SIMULATED_SensorDetectedObject_H_
+#define INCLUDE_SIMULATED_SensorDetectedObject_H_
+
+#include <tmx/messages/message.hpp>
+#include <MessageTypes.h>
+#include <Vector3d.h>
+#include <Point.h>
+
+namespace tmx
+{
+    namespace messages
+    {
+       
+        /**
+         * This SensorDetectedObject is used to communicate the sensor detected object information with various applications
+         * including internal infrastructure applications and external road user applications through simulated environment.
```

Comment:
Which specific infrastructure applications and external road user applications? It seems this only used internally by V2xHub CDASimAdapter and CARMA Streets sensor data sharing service.

Reply:
I do not think the message should document which use cases use it. I think the message is agnostic to use case and the users of the message should document which messages they use. Otherwise every use case would be updating generic messages to link back to their new use case. Please let me know if I misunderstood you comment.

Reply:
I meant literally `including internal infrastructure applications and external road user applications through simulated environment`. If I am an external user, reading this, and not sure the definition of infrastructure apps & road user apps. If this is used by MUST device in real world as well. It is partially correct that the communication is through simulation environment without mentioning real work sensors.

Reply:
I have removed application information from the message description